src/aliases.js

```diff
@@ -0,0 +1,31 @@
+import { promises as fs } from 'fs'
+import { version } from 'process'
+
+import findUp from 'find-up'
+
+export const NODE_VERSION_ALIAS = '.'
+export const NODE_VERSION_FILES = ['.node-version', '.nvmrc', '.naverc']
+
+export const CURRENT_NODE_ALIAS = '_'
+
+export const resolveVersionRangeAlias = async function (
+  versionRange,
+  { cwd } = {},
+) {
+  if (versionRange === CURRENT_NODE_ALIAS) return version
+
+  if (versionRange !== NODE_VERSION_ALIAS) return versionRange
+
```

Comment:
Could you please put the `NODE_VERSION_ALIAS` in its own function. Like this it will be easier to add new aliases.